Firestore/core/src/firebase/firestore/remote/serializer.cc

```diff
@@ -141,7 +141,7 @@ google_firestore_v1_MapValue EncodeMapValue(
 
   size_t count = object_value_map.size();
 
-  result.fields_count = count;
+  result.fields_count = static_cast<pb_size_t>(count);
```

Comment:
I'll assume this is intentional.

Reply:
Yes. There was a separate commit for these.